previousnext/agov
# WIP: 34187 Header

Once https://github.com/previousnext/agov/pull/649 goes in this commits below will be cleaner if I open/close this PR

Steps to test: 

Task #34769: Logo
Make sure correct logo is displaying. If we have SVG, this would be preferable, currently using PNG. We should be able to swap out logo by going to theme settings.

Task #34771: Rebuild styleguide version of search with correct markup documentation	
Make sure that the search component is demonstrated using dream markup in the styleguide (not Drupla markup). Test on both smaller and larger screens and with screenreader.

Task #34772: Default search block placement via ago install profile
The search block should be placed in the header region here https://agov-pr650.agov.com.au/
		
Task #34773: Theming of search block to match styleguide
The search block should look like what is in the styleguide, check that the Drupal markup is as close as we can make it to what is in the styleguide and that template files look OK, I am stripping everything we don't need out. One notable difference to agov3 is there are two search forms hiding/showing in D7 version... now CSS just places the form where it needs to go.
	
Task #34779: Theme and place user menu to header in Drupal
Make sure user menu is placed and is themed properly.

Task #34792: Default menu link (content) in user menu
Currently the Sitemap link goes to the homepage because the sitemap doesn't exist yet. You can make changes to https://github.com/previousnext/agov/blob/34187-header/agov.links.menu.yml If you want to edit this content.
